src/h5web/storage-utils.ts

```diff
@@ -21,7 +21,7 @@ function fromStorage<T>(id: string): T | undefined {
 export function createPersistableState<S extends State>(
   config: StorageConfig,
   createState: StateCreator<S>
-): [UseStore<S>, StoreApi<S>] {
+): UseStore<S> {
```

Comment:
The return type of zustand's `create` function has changed. It's now the store's hook on its own.